urbanbroccoli/sor.py

```diff
@@ -31,13 +31,13 @@ def sor_1d(rho, hx, epsilon, maxiter, maxerr, w):
     return phi
 
 def sor_2d(rho, hx, hy, epsilon, maxiter, maxerr, w):
-    """Performs a sucsesive over relaxation for the poisson equation of a two dimensional discretized charge distribution:
+    """Performs a sucsessive over-relaxation for the Poisson equation of a two dimensional discretized charge distribution:
           Parameters:
```

Comment:
This line has an indention level of 10 spaces. Indentions should be multiples of four and, most important, they should be consistent. Use an indention level of four spaces, here.